packages/framework/src/Model/Product/ProductOnCurrentDomainElasticFacade.php

```diff
@@ -206,11 +227,16 @@ public function getPaginatedProductsForSearch($searchText, ProductFilterData $pr
      * @param string $orderingModeId
      * @param int $page
      * @param int $limit
-     * @param string $searchText
+     * @param string|null $searchText
      * @return \Shopsys\FrameworkBundle\Model\Product\Search\FilterQuery
      */
-    protected function createListableProductsForSearchTextFilterQuery(ProductFilterData $productFilterData, $orderingModeId, $page, $limit, $searchText): FilterQuery
-    {
+    protected function createListableProductsForSearchTextFilterQuery(
+        ProductFilterData $productFilterData,
+        string $orderingModeId,
+        int $page,
+        int $limit,
+        ?string $searchText
```

Comment:
It's still nullable? `getPaginatedProductsForSearch` was changed to not-null

Reply:
Yes it can be because this is used for autocompleting as well and the search text there could be nullable.

Reply:
👌